lib/ansible/modules/packaging/os/swupd.py

```diff
@@ -0,0 +1,288 @@
+
+    if FILES_NOT_MATCH in stdout:
+        return True
+
+    return False
+
+
+def swupd_install(module, bundle):
+    """Installs a bundle with `swupd bundle-add bundle`"""
+    if module.check_mode:
+        module.exit_json(changed=not _is_bundle_installed(module, bundle))
+
+    if _is_bundle_installed(module, bundle):
+        module.exit_json(changed=False, msg="Bundle %s is already installed" % bundle)
+
+    cmd = _get_cmd(module, "bundle-add %s" % bundle)
+    rc, stdout, stderr = module.run_command(cmd, check_rc=False)
+
+    if rc == 0:
+        module.exit_json(changed=True, msg="Bundle %s installed" % bundle, stdout=stdout, stderr=stderr)
+
+    if rc == 18:
+        module.exit_json(Changed=False, msg="Bundle name %s is invalid" % bundle, stdout=stdout, stderr=stderr)
+
+    module.fail_json(msg="Unkown Error", stdout=stdout, stderr=stderr)
+
+
+def swupd_remove(module, bundle):
+    """Removes a bundle with `swupd bundle-remove bundle`"""
+    if module.check_mode:
+        module.exit_json(changed=_is_bundle_installed(module, bundle))
+
+    if not _is_bundle_installed(module, bundle):
+        module.exit_json(changed=False, msg="Bundle %s not installed" % bundle)
+
+    cmd = _get_cmd(module, "bundle-remove %s" % bundle)
+    rc, stdout, stderr = module.run_command(cmd, check_rc=False)
+
+    if rc == 0:
+        module.exit_json(changed=True, msg="Bundle %s removed" % bundle, stdout=stdout, stderr=stderr)
+
+    module.fail_json(msg="Unkown Error", stdout=stdout, stderr=stderr)
+
+
+def swupd_update(module):
+    """Updates the os with `swupd update`"""
+    if module.check_mode:
+        module.exit_json(changed=_needs_update(module))
+
+    if not _needs_update(module):
+        module.exit_json(changed=False, msg="There are no updates available")
+
+    cmd = _get_cmd(module, "update")
+    rc, stdout, stderr = module.run_command(cmd, check_rc=False)
+
+    if rc == 0:
+        module.exit_json(changed=True, msg="Update successful", stdout=stdout, stderr=stderr)
+
+    module.fail_json(msg="Unkown Error", stdout=stdout, stderr=stderr)
+
+
+def swupd_verify(module):
+    """Verifies filesystem agains specified or current version"""
+    if module.check_mode:
+        module.exit_json(changed=_needs_verify(module))
+
+    if not _needs_verify(module):
+        module.exit_json(changed=False, msg="No files where changed")
+
+    cmd = _get_cmd(module, "verify --fix")
+    rc, stdout, stderr = module.run_command(cmd, check_rc=False)
+
+    if rc == 0 and (FILES_REPLACED in stdout or FILES_FIXED in stdout or FILES_DELETED in stdout):
+        module.exit_json(changed=True, msg="Fix successful", stdout=stdout, stderr=stderr)
+
+    module.fail_json(msg="Unkown Error", stdout=stdout, stderr=stderr)
+
+
+def main():
+    """The main function."""
+    module = AnsibleModule(
+        argument_spec=dict(
+            contenturl=dict(default=None, type='str'),
```

Comment:
The `default=None` can be removed. And I think the `type=str` can be removed as well as it's the default type.